src/Eris/Generator/AssociativeArrayGenerator.php

```diff
@@ -14,7 +14,12 @@ function associative(array $generators)
 class AssociativeArrayGenerator implements Generator
 {
     private $generators;
-    
+
+    /**
```

Comment:
for all the code in `src` fields are welcome and their absence means I forgot. Going to the root cause again, I can never configure PHP to error out when they are missing, is there some kind of static analysis we can run on src/ to make sure they are always there?